modules/ztp-install-factory-pipeline-overview.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="factory-install-workflow_{context}"]
+= The factory install workflow
+
+The factory install pipeline builds out your factory environment in readiness for the edge cluster to be shipped off site.
+
+The following diagram provides a high level overview of the pipelines used to prepare the edge clusters:
+
+.Hub and edge pipelines
+
+image::225_OpenShift_Installing_Clusters_0422_pipeline.png[Factory install workflow]
```

Comment:
The diagram shows UI and day2 worker installed in parallel. We see the worker being installed after UI.

Reply:
I will make that note - was knida mentioned in video as being parallel and pipeline in OpenShift UI pipelines indicates in parallel but you tested so let me look at again. Maybe a note would suffice just to make it clear.